impl/src/main/java/org/glassfish/json/JsonParserFactoryImpl.java

```diff
@@ -41,17 +41,17 @@
 
     @Override
     public JsonParser createParser(Reader reader) {
-        return new JsonParserImpl(reader, bufferPool);
+        return new JsonParserImpl(reader, bufferPool, true);
     }
 
     @Override
     public JsonParser createParser(InputStream in) {
-        return new JsonParserImpl(in, bufferPool);
+        return new JsonParserImpl(in, bufferPool, true);
     }
 
     @Override
     public JsonParser createParser(InputStream in, Charset charset) {
-        return new JsonParserImpl(in, charset, bufferPool);
+        return new JsonParserImpl(in, charset, bufferPool, true);
```

Comment:
Why not just adding new constructor and redirecting existing one to it, so the rest of the code does not have to be updated ?

Reply:
I made the changes that you have recommended. Please see my latest commit.